### PR TITLE
Allow restriction of tournaments on results screens

### PR DESCRIPTION
**Pull request by Amaras**
Improvement of results screens mentioned in #27: only include specified tournaments in a screen of type `results`.

This is untested for now (coded on Linux), as the query can be quite problematic.

Documentation was not modified to reflect the change yet.

### CODE REVIEW

**Comment by Amaras** [comment]
This feature is now implemented as I intended and is documented.
It should be able to be merged, awaiting a review.

This use case is probably rare, but at least there's an example of how to do something like this in the future.

File: database/sqlite.py [hunk 1]
```diff
@@ -206,13 +206,36 @@ def delete_result(self, tournament_id: str, round: int, board_id: int):
...
+                                   'FROM `result` WHERE `tournament_id` = ? ORDER BY `date` DESC')
+                params += (tournament, )
+                if limit:
+                    query_part += ' LIMIT ?'
+                    params += (limit, )
+                query_parts.append('\n'.join(('SELECT * FROM (', query_part, ')')))
+            query: str = '\nUNION\n'.join(query_parts) + ' ORDER BY `date` DESC'
+            if limit:
+                query += ' LIMIT ?'
+                params += (limit, )
```
Comment (Amaras) [inline database/sqlite.py:239]: Check if `IN (?, ?, ...)` pattern is possible in SQLITE prepared statements

File: docs/24-last-results.md [hunk 2]
```diff
@@ -19,8 +19,21 @@ limit = 30
 
 Le paramètre `limit` permet de limiter le nombre de résultats affichés (ici 30).
 
+## Limitation des tournois dans les écrans de résultats
+
+Dans le cas où un écran de résultats est affiché aux spectateurs, il peut être appréciable de scinder cet écran pour ne montrer que les résultats d'un ou plusieurs tournois. Pour un événement comprenant 3 tournois (`a`, `b` et `jeunes`), il est possible de définir deux écrans de résultats, un nommé `résultats-tc` (tournois `a` et `b`), et un autre nommé `résultats-j` (tournoi jeunes) de cette manière :
```
Comment (Amaras) [inline docs/24-last-results.md]:
```suggestion
Dans le cas où un écran de résultats est affiché aux spectateurs, il peut être appréciable de scinder cet écran pour ne montrer que les résultats d'un ou plusieurs tournois. Par exemple, pour un événement comprenant 3 tournois (`a`, `b` et `jeunes`), il est possible de définir deux écrans de résultats, un nommé `résultats-tc` (tournois `a` et `b`), et un autre nommé `résultats-j` (tournoi jeunes) de cette manière :
```

**Review by Amaras — COMMENTED** [review]
Before review

**Review by pascalaubry — APPROVED** [review]

**Pull request merged**